data-prepper-plugins/common/README.md

```diff
@@ -8,9 +8,27 @@ A prepper plugin to generate new string records with upper or lower case convers
 
 ## `file` (source)
 
-A source plugin to read input data from the specified file path.
-
-- path (String): absolute input data file path
+A source plugin to read input data from the specified file path. The file source creates a new Record for eadh line of data in the file.
+
+* `path` (String): absolute input data file path. It is required
+  
+* `write_timeout` (int): The amount of time to attempt writing a Record to the Buffer before timing out. Unit is milliseconds and default is `5,000`  
+  
+* `format` (String): The format of each line of the file. Valid options are `json` or `plain`. Default is `plain`.
+  <br></br>
+    * `plain`: Reads plaintext data from files. Internally, a plain text line from a file will be given a key of `message` as shown below.
+    ```
+    Example log line in file
+    ```
+  becomes 
+    ```
+    { "message": "Example log line in file" }
+    ```
+  
+    * `json`: Reads data that is in the form of a JSON string from a file. If the json string is unable to be parsed, the file source will treat it as a plaintext line.
```

Comment:
From the code change, maybe we should specify that this format expects json lines as follows
```
{ "key1": "val1" }
{ "key2": "val2" }
{ "key3": "val3" }
```
instead of a single json blob